charts/scatterCharts/ScatterTransform.ts

```diff
@@ -389,10 +389,14 @@ export class ScatterTransform extends ChartTransform {
         const values = this.allPoints.filter(
             point => point.x !== 0 && point.y !== 0
         )
+
         for (let i = 0; i < values.length; i++) {
             const indexValue = values[i]
             for (let j = i; j < values.length; j++) {
                 const targetValue = values[j]
+
+                if (targetValue.entityName !== indexValue.entityName) continue
```

Comment:
Basically the only reason I'm putting this up as a PR and not simply pushing this is:
From its type definition, `ScatterValue.entityName` can be `undefined`.
When could this be the case, and do we need to take special care for that case?

cc @breck7 maybe

Reply:
I don't think we would have have that as undefined. It looks to me like we could just update the type signature and line ~724 in ScatterTransform to provide an entityName.